tests/foreman/cli/test_import.py

```diff
@@ -273,6 +273,62 @@ def test_reimport_orgs_default_negative(self, test_data):
             'rm -rf ${{HOME}}/.transition_data {}'.format(files['users'])
         )
 
+    @data(*import_org_data())
+    def test_import_orgs_recovery(self, test_data):
+        """@test: Try to Import organizations with the same name to invoke
+        usage of a recovery strategy (rename, map, none)
+
+        @feature: Import Organizations Recover
+
+        @assert: 2nd Import will rename the new organizations, 2nd import will
+        map them and the 3rd one will resul in No Action Taken
+
+        """
+        # prepare the data
+        files = dict(self.default_dataset[1])
+        new_dataset = update_csv_values(files['users'], test_data)
+        ssh.upload_file(new_dataset, new_dataset)
+        files['users'] = new_dataset
+        # initial import
+        self.assertEqual(
+            Import.organization({'csv-file': files['users']}).return_code, 0)
+        # clear the .transition_data to clear the transition mapping
+        ssh.command('rm -rf ${HOME}/.transition_data')
+        # use the default (rename) strategy
+        self.assertIn(
```

Comment:
I completely support idea of "or compare the entity listing before and after performing the import (which is slower as it requires performing 2 extra hammer commands).". Could we measure in some way what impact we will have in case of such implementation?